include/MainWindow.h

```diff
@@ -81,9 +82,25 @@ class MainWindow : public QMainWindow
 	///
 	bool mayChangeProject(bool stopPlayback);
 
-	void autoSaveTimerStart()
+	// Auto save timer intervals
+	static const int DEFAULT_AUTO_SAVE_INTERVAL = 2;
+	int m_autoSaveInterval;
+	const static int m_autoSaveShortTime = 10 * 1000; // 10s short loop
+
+	void autoSaveTimerReset( int msec = ConfigManager::inst()->
+					value( "ui", "saveinterval" ).toInt()
+						* 60 * 1000 )
+	{
+		if( msec < 60 * 1000 || msec > 20 * 60 *1000 )
+		{
+			msec = DEFAULT_AUTO_SAVE_INTERVAL * 60 * 1000;
```

Comment:
If autosave (`msec`) is less than 1 minute or greater than 20 minutes, set it to 2 minutes?  Why exactly?